technologies/app/saagie-hf-modelserver-textclf/metadata.yaml

```diff
@@ -0,0 +1,25 @@
+version: v2
+type: APP
+id: saagie-hf-modelserver-textclf
+label: Saagie HF ModelServer TextCLF
+baseline: "Saagie HuggingFace Model Server for Text Classification"
+description: "Saagie-HF-ModelServer-TextCLF: Custom app based on Dash/Flask that deploys the deep learning models from HuggingFace and makes predictions for text classification via the GUI or API"
+available: true
+icon: datascience
+backgroundColor: "#253c5b"
+customFlags: []
+contexts:
+  - id: saagie-hf-modelserver-textclf-0.1
+    label: For Saagie V2
+    deprecationDate: "2023-08-01T00:00:00Z"
```

Comment:
Maybe the deprecationDate is not necessary, because we'll have something like this : 
<img width="757" alt="deprecated_banner" src="https://github.com/saagie/technologies-community/assets/60429863/3249ad38-ce10-4e3e-aa14-36f369adfa8e">